.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: "📕 Documentation Issue"
```

Comment:
Do we even have real documentation for this?

Reply:
The README can always be improved! Thought about not including this but I don't see the harm in including it if it's probably going to be useful at some point